backend/tests/v5/helper/services.js

```diff
@@ -95,6 +96,13 @@ db.createRevision = (teamspace, modelId, revision) => {
 	return DbHandler.insertOne(teamspace, `${modelId}.history`, formattedRevision);
 };
 
+db.createRevisionRef = async (teamspace, modelId, refId) => {
+	const data = 'test data';
+	let refInfo = await ExternalServices.storeFile(teamspace, `${modelId}.history.ref`, data);
```

Comment:
We need to make sure we clean the folder after the tests or we end up with clutter, or files that shouldn't exist to start existing.

the fileshare is set here https://github.com/3drepo/3drepo.io/blob/master/config/testV5/config.js#L82

So as part of the tear down (the bit that is called in [`afterAll` in jest](https://jestjs.io/docs/setup-teardown#one-time-setup)), you will probabyl want a function to remove all contents within that folder.